alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -156,6 +166,111 @@ def listdir(self, path):
                 f"Error when listing path {path}: error {e}"
             ) from e
 
+    def load(
+        self,
+        file_path,
+        timeout=None,
+    ):
+        """
+        Loads a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+            timout (integer): The number of seconds for timeout
+
+        Returns:
+            result (boolean): Whether the file has been loaded successfully
+        """
+        worker_host = self._get_preferred_worker_host(file_path)
+        return self._load_file(worker_host, file_path, timeout)
+
+    def submit_load(
+        self,
+        file_path,
+    ):
+        """
+        Submits a load job for a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+
+        Returns:
+            result (boolean): Whether the job has been submitted successfully
+        """
+        worker_host = self._get_preferred_worker_host(file_path)
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=file_path,
+                ),
+            )
+            response.raise_for_status()
+            return b"successfully" in response.content
+        except Exception as e:
+            raise Exception(
+                f"Error when submitting load job for path {file_path} from {worker_host}: error {e}"
+            ) from e
+
+    def stop_load(
+        self,
+        file_path,
+    ):
+        """
+        Stops a load job for a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+
+        Returns:
+            result (boolean): Whether the job has been stopped successfully
+        """
+        worker_host = self._get_preferred_worker_host(file_path)
+        try:
+            response = self.session.get(
+                self.LOAD_STOP_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=file_path,
+                ),
+            )
+            response.raise_for_status()
+            return b"successfully" in response.content
+        except Exception as e:
+            raise Exception(
+                f"Error when stopping load job for path {file_path} from {worker_host}: error {e}"
+            ) from e
+
+    def load_progress(
+        self,
+        file_path,
+    ):
+        """
+        Gets the progress of the load job for a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+
+        Returns:
+            result (str): The progress of the load job
```

Comment:
is it possible to return a struct instead of a string which is hard to know which is the allowed value and what to expect

Reply:
It's kinda of not easy to do so.

Reply:
@ssyssy Please help change it to a struct 
e.g. SUCCEED, FAILED or INPROGRESS, by passing the RESTful API results in the python API